engine/runtime/src/main/java/org/enso/interpreter/runtime/HostClassLoader.java

```diff
@@ -31,7 +31,13 @@ final class HostClassLoader extends URLClassLoader implements AutoCloseable {
   // boot module layer.
   private static final boolean isRuntimeModInBootLayer;
 
-  public HostClassLoader() {
+  private static final HostClassLoader INSTANCE = new HostClassLoader();
+
+  public static HostClassLoader getClassLoader() {
```

Comment:
- there are various attempts to rewrite this `HostClassLoader`
- one of them is #10714
- it attempts to provide [independent classloader for each library](https://github.com/enso-org/enso/pull/10714/files#r2131518443)
- making `HostClassLoader` a singleton goes _against such efforts_
- but it may be good for purposes of this PR 
- I guess we can unsingletonize the classloader later